source/documentation/services/airflow/index.md

```diff
@@ -20,19 +24,21 @@ Our Kubernetes infrastructure is connected to the MoJO Transit Gateway, which co
 
 If you need additional connectivity, [submit a feature request](https://github.com/ministryofjustice/analytical-platform/issues/new?template=feature-request-template.yml).
 
-> **Please note**: You cannot use Analytical Platform Airflow for pipelines using `BashOperator` or `PythonOperator`
+> **Please note**: You cannot use Analytical Platform Airflow for workflows using `BashOperator` or `PythonOperator`
 
 ## Concepts
 
-We organise Airflow pipelines using **environments**, **projects** and **workflows**:
+![](images/airflow/airflow_diagram.png)
```

Comment:
Added the following image as I am very much a visual person

![airflow_diagram](https://github.com/user-attachments/assets/472aef15-8be0-4dc2-94b2-e95428aa49a4)

Reply:
I like these but how can I edit it? I want to lowercase the project and change manifest.yml to workflow.yml

Reply:
I'd also recommend having this as an SVG rather than PNG - better format type for resizing and accessibility.

Reply:
Can't seem to share the excalidraw file on here. Will make the changes now and save as SVG